src/content/docs/merge-queue/setup.mdx

```diff
@@ -141,13 +141,15 @@ queues to your specific needs.
 For further exploration and inspiration, here are some additional resources and
 use cases that you might find useful:
 
-- Using [priorities to order your pull requests](priority);
-- Using [multiple queues](multi);
-- Speeding up your merge queue with [speculative checks](speculative-checks);
-- Saving CI time with [batches](batches);
+- Using [priorities to order your pull requests](/merge-queue/priority);
+- Using [multiple queues](/merge-queue/multi);
+- Speeding up your merge queue with [speculative checks](/merge-queue/speculative-checks);
+- Saving CI time with [batches](/merge-queue/batches);
 {/* FIXME: add link */}
 - Delaying checks at merge time;
-- Splitting repositories into [partitions](partitions) (monorepo support).
+
```

Comment:
```suggestion
```

Reply:
This one is a eslint rule, need one blank line because the item is now two line wide